### DISCUSSION TITLE
ファイヤーフォックス

### DISCUSSION THREAD

**Discussion by 08077199933** [mozilla-mobile/firefox-ios#11182] [General]
[![image](https://user-images.githubusercontent.com/106266868/176992876-f686b448-332c-48f8-a0cc-311262819315.jpeg)]()

**Comment by 08077199933** [comment]
- **_[![image](https://user-images.githubusercontent.com/106266868/176992911-d5dfa17d-d800-463f-b197-aa6f1525e4c5.jpeg)]()_**

**Comment by lmarceau** [comment]
Hello! Would you mind explaining in English what's the issue/question?
